### DISCUSSION TITLE
Is there either an alternative to or a way to make List views horizontal instead of vertical?

### DISCUSSION THREAD

**Discussion by evanAndersonDigipen** [tui-cs/Terminal.Gui#1086] [Q&A]
ive got a horizontal program with a frame view to activate beneath it, sorta like the UICatalog example but rotated, is there a way to make List views horizontal like buttons kinda or is there an alternative to it?

**Comment by tznind** [comment]
I know it's been a while but you might want to look into  `TabView`.  Sorta sounds like what you are looking for? I think it is in the current prerelease version and should be in 1.0.0

https://github.com/migueldeicaza/gui.cs/blob/main/Terminal.Gui/Views/TabView.cs

![image](https://user-images.githubusercontent.com/31306100/115368719-4269fd80-a1bf-11eb-9f55-88815a12f002.png)
